### DISCUSSION TITLE
InputKit Official Documentation 📃

### DISCUSSION THREAD

**Discussion by enisn** [enisn/Xamarin.Forms.InputKit#282] [Announcements]
InputKit documentation is moved to the following link. 🥳🎊
https://enisn-projects.io/docs/en/inputkit/

All the existing partial documents are merged in there.
From now, further documentation updates will be there.

**Comment by Nerves82** [comment]
Is there any plan to update the documentation? It seems wildly out of date. For example, on the advanced entry there is no longer a MinLength property or AnnotationMessage property. It would also be great to get some documentation on how validations are handled.
